### PR TITLE
Adds ability to set custom task lifetime

### LINKED ISSUE #778: [Feature Request] Set task expires time
How can I set the task expiration time to one week? Currently, tasks will be expired in one month. It's too long.

### PR DESCRIPTION
**Pull request by Etesam913**
Should resolve #778

### CODE REVIEW

File: mephisto/abstractions/providers/mturk/mturk_utils.py [hunk 1]
```diff
@@ -506,6 +506,7 @@ def create_compensation_hit_with_hit_type(
     is_sandbox = client_is_sandbox(client)
 
     # Create the HIT
+    # Question: Does method below need to be updated as well?
     response = client.create_hit_with_hit_type(
         HITTypeId=hit_type_id,
```
Comment (Etesam913) [inline mephisto/abstractions/providers/mturk/mturk_utils.py:511]: Does this parameter need to be updated as well

File: mephisto/abstractions/providers/mturk/mturk_utils.py [hunk 2]
```diff
@@ -506,6 +506,7 @@ def create_compensation_hit_with_hit_type(
     is_sandbox = client_is_sandbox(client)
 
     # Create the HIT
+    # Question: Does method below need to be updated as well?
```
Comment (Etesam913) [inline mephisto/abstractions/providers/mturk/mturk_utils.py:509]: question

Comment (JackUrb) [reply mephisto/abstractions/providers/mturk/mturk_utils.py:509]: Good question - not this one. Here we're just putting up a static task for a worker to complete on their own time, expiration here is essentially meaningless.

Would be worthwhile to include the answer in the comment
```
# Create a compensation HIT to be completed in the next month
```

**Review by JackUrb — APPROVED** [review]
Thanks for this fix @Etesam913! I know it will be useful for many, especially to help alleviate clog for those who don't run `cleanup.py` often.

**Pull request merged**